addons/api/mirage/scenarios/ipc.js

```diff
@@ -199,6 +199,18 @@ export default function initializeMockIPC(server, config) {
     getDesktopVersion() {
       return { desktopVersion: '1.0.0' };
     }
+    getLogLevel() {
+      return 'info';
+    }
+    getLogPath() {
+      return '~/.config/Boundary/logs/desktop-client.log';
+    }
+    setLogLevel() {}
```

Comment:
just a heads up - I added this in my PR as there was a comment about console errors

Reply:
Yup no problem, I'll merge any issues